epub34/authoring/index.html

```diff
@@ -1002,6 +1002,13 @@ <h3>Core media types</h3>
 						<tr>
 							<th colspan="3" id="cmt-grp-image" class="tbl-group">Images</th>
 						</tr>
+						<tr>
+							<td id="cmt-avif" data-tests="#pub-cmt-avif">
+								<code>image/avif</code>
+							</td>
+							<td> [[av1-avif]] </td>
+							<td>AVIF Images</td>
```

Comment:
"Images" doesn't need to be capitalized. We have a weird mix of caps in the descriptions, none of which are necessary. They should all be sentence case for consistency. Do you want to clean up the others or do you want me to?

```suggestion
							<td>AVIF images</td>
```

Reply:
I will do it, @mattgarrish

Reply:
Done in https://github.com/w3c/epub-specs/pull/2811/commits/4291489b3fadd0587150de332cc9e0b1745bec32